daemon/mgr/spec_linux.go

```diff
@@ -55,3 +57,24 @@ func setupAppArmor(ctx context.Context, meta *ContainerMeta, spec *SpecWrapper)
 
 	return nil
 }
+
+func setupCapabilities(ctx context.Context, meta *ContainerMeta, spec *SpecWrapper) error {
+	var caplist []string
+	var err error
+
+	s := spec.s
+	if meta.HostConfig.Privileged {
```

Comment:
Could you add this flag in pouch command line as well? 
I found that in cli, there is no privileged flag. @YaoZengzeng

Reply:
Actually we could see meta.HostConfig.Privileged in many places associated with security.

I'll use an independent PR to handle it once the security options are complete.